easybuild/tools/options.py

```diff
@@ -66,7 +67,9 @@
 
 
 XDG_CONFIG_HOME = os.environ.get('XDG_CONFIG_HOME', os.path.join(os.path.expanduser('~'), ".config"))
-DEFAULT_CONFIGFILE = os.path.join(XDG_CONFIG_HOME, 'easybuild', 'config.cfg')
+XDG_CONFIG_DIRS = os.environ.get('XDG_CONFIG_DIRS', os.path.join("/etc"))
```

Comment:
yes it can be: 
```
$XDG_CONFIG_DIRS defines the preference-ordered set of base directories to search for configuration files in addition to the $XDG_CONFIG_HOME base directory. The directories in $XDG_CONFIG_DIRS should be seperated with a colon ':'.
```
So the glob on the next line should be fixed to be `DEFAULT_SYSTEM_CONFIGFILES = [(y for y in [os.path.join(x, 'easybuild.d', '*.cfg') for x in XDG_CONFIG_DIRS.split(':')]]`